packages/cozy-client/src/RealTimeQueries.jsx

```diff
@@ -55,11 +57,12 @@ const dispatchChange = (
  * Component that subscribes to a doctype changes and keep the
  * internal store updated.
  *
- * @param  {object} options - Options
- * @param  {import("./types").Doctype} options.doctype - The doctype to watch
+ * @param {object} options - Options
+ * @param {import("./types").Doctype} options.doctype - The doctype to watch
+ * @param {Function} options.customNormalization - Function to call before dispatching the change
  * @returns {null} The component does not display anything.
  */
-const RealTimeQueries = ({ doctype }) => {
+const RealTimeQueries = ({ doctype, customNormalization }) => {
```

Comment:
Nice! Maybe we should have the ability to customize the "normalization" by event (create / update / delete). 

Also, I would have done that outside of the `normalization` step, because here it's not a normalization anymore, it's a "business logical" before dispatching the result. 

I didn't have a good naming yet, but : 
`customComputeDocumentFromRealtime` can be more appropriated? 

In fact, we already have this `computeDocumentFromRealtime` without knowing it... : 
```
{ ...couchDBDoc, _deleted: true },
```

So this one cas ben the default on deleteEvent.